Engine/src/Engine/MainMenu.cpp

```diff
@@ -0,0 +1,73 @@
+#include "MainMenu.h"
+
+void MainMenu::show()
+{
+	if (ImGui::BeginMainMenuBar())
+	{
+		//sets the font colour to be white
+		s_style->Colors[ImGuiCol_Text] = white;
+
+		//Do not use else-if statements here otherwise
+		//the menu titles will not render correctly
+		//creates a drop down menu where if x is true (open), display y
+		if (ImGui::BeginMenu("File"))
+		{
+			if (ImGui::MenuItem("New", nullptr, nullptr))
+			{
+				//Handle menu item...
```

Comment:
I know this might be in process already, but it would be good to have a Trello ticket to keep track of these handles